### PR TITLE
[Modular] [Ready] Adds some new traitor items

### PR DESCRIPTION
**Pull request by lyricalpaws**
## About The Pull Request
Does what title says, Adds some new bundles.

They are: Ocelot, Doctor Death, Donk Co. Shill, The Downtown Special.

Also adds some new items.

Nuclear Delivery Grenade, Box of Extra Large Shot Glasses

Also removes the Elite Hardsuit from rotation.

## Why It's Good For The Game
Replace/Improve over removal is what xyel told me, so I did so for the Silverhand and Elite suit's removal.

## Changelog
:cl:
add: Traitor items
del: removed Elite Suit from Uplink
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
Would be nice to have the bundles described for people that can't read code

**Comment by lyricalpaws** [comment]
^ Restricting shot glasses to bartendo

**Comment by lyricalpaws** [comment]
> 
> 
> Would be nice to have the bundles described for people that can't read code

Will Write up a list when I have the chance!

File: modular_skyrat/master_files/code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -233,12 +249,12 @@
...
-	desc = "An upgraded, elite version of the Syndicate hardsuit. It features fireproofing, and also \
-			provides the user with superior armor and mobility compared to the standard Syndicate hardsuit."
-	item = /obj/item/clothing/suit/space/hardsuit/syndi/elite
-	cost = 10
+///datum/uplink_item/suits/hardsuit/elite_traitor
+//	name = "Elite Syndicate Hardsuit"
+//	desc = "An upgraded, elite version of the Syndicate hardsuit. It features fireproofing, and also \
+//			provides the user with superior armor and mobility compared to the standard Syndicate hardsuit."
+//	item = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+//	cost = 10
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/uplink/uplink_items.dm]: This is not how you comment out something. Put a `/*` at the beginning and a `*/` at the end. Plus, this is a modular file so you can just delete it, no reason to leave it commented.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	desc = "The guy wearing this means serious business..."
+	armor = list(MELEE = 25, BULLET = 20, LASER = 30, ENERGY = 30, BOMB = 30, BIO = 0, RAD = 10, FIRE = 30, ACID = 30, WOUND = 20)//Slightly worse than the sec coat
+
+/obj/item/clothing/under/suit/blacktwopiece/armoured
+	armor = list(MELEE = 10, BULLET = 10, LASER = 10,ENERGY = 10, BOMB = 0, BIO = 0, RAD = 10, FIRE = 50, ACID = 40)//Same as tacticool
+
+/obj/item/clothing/head/beret/sec/ocelot
+	name = "ocelot beret"
+	desc = "A red beret with the neo-russian insignia stamped on it. This bitch is wearing perfume."
+	armor = list(MELEE = 30, BULLET = 20, LASER = 20, ENERGY = 30, BOMB = 25, BIO = 0, RAD = 0, FIRE = 20, ACID = 50, WOUND = 4)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm:12]: Berets are refactored upstream by yours truly, so just keep that in mind, if you want it to look different.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass/syndicate
+	name = "shot glass"
+	desc = "A shot glass - the universal symbol for terrible decisions."
+	icon_state = "shotglass"
+	base_icon_state = "shotglass"
+	gulp_size = 50
+	amount_per_transfer_from_this = 50
+	possible_transfer_amounts = list(50)
+	volume = 50
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm:9]: Not sure how I feel about this, it just feels like something that wouldn't really lead to a lot of interaction with the players, 50u is a really big amount for something so stealthy, although, to be fair, don't they tell how much liquid they contain?

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm (hunk 2 above)
Comment (lyricalpaws) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm:12]: This was done pre mirror, and it takes the icon from the sec beret so it'll change when the overhaul is done.

**Comment by SpaceVampire** [comment]
The Nuclear Delivery Grenade is probably under-powered/overpriced.
Compared to the Viscerator Delivery Grenade, that already exists.
Plus you have golden core spam for the same general stuff.
The .357 peacemaker bullet should most likely have the normal wound bonus.
Given that we made the default on all bullets 20, so I would just remove the line making it zero.
Just my two cents after thinking about it for about a minute.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm (hunk 2 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm:12]: > This was done pre mirror, and it takes the icon from the sec beret so it'll change when the overhaul is done.

Yuh, I was just saying, if you need to know how to do it, just poke me on Discord.

**Comment by lyricalpaws** [comment]
> 
> 
> The Nuclear Delivery Grenade is probably under-powered/overpriced.
> Compared to the Viscerator Delivery Grenade, that already exists.
> Plus you have golden core spam for the same general stuff.
> The .357 peacemaker bullet should most likely have the normal wound bonus.
> Given that we made the default on all bullets 20, so I would just remove the line making it zero.
> Just my two cents after thinking about it for about a minute.

the 375 has a negative wound bonus. I made it 0 as I reduced the damage from 60 to 25. As for the nuke delivery grenade, I kept toying with the idea. Any changes would be appreciated. Think on these are the blacksuits (or at least should be).

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm (hunk 3 above)
Comment (lyricalpaws) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm:9]: They shouldn't say, no. They're mostly a thing I've wanted to bring from goon for a really long time, and could lead to some interesting bartending shinanigans.

File: modular_skyrat/master_files/code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (lyricalpaws) [reply modular_skyrat/master_files/code/modules/uplink/uplink_items.dm]: Deleted it, same with robohand. The code for the item and bundle are still in the game, but I don't plan to bring them back so.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm (hunk 3 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/drinks/drinkingglass.dm:9]: I just have a feeling that it won't really contribute much to roleplay, someone will eventually put a very deadly mix into those and still have space for some regular booze so people wouldn't expect it and would get like 20u of a very deadly mix of chemicals. I don't know how good that'd be for roleplay, honestly.

**Comment by lyricalpaws** [comment]
Properly hides the reagent amount (Still sus but not as sus)

**Comment by lyricalpaws** [comment]
> 
> 
> Would be nice to have the bundles described for people that can't read code

Ocelot Bundle: Custom Revolver, Ammo, Armoured trenchcoat and beret, Chameleon Holster

Downtown Special: Tommygun, Ammo, Armoured jacket and suit, Whiskey, sunglasses

Donk shill: Riot Darts, Donk guns, Donk pockets, Donksoft vendor + refills, toolbox

Doctor Death: Poison kit, Dart pistol, Metal foam, syndie soap, and carp clusters, toolbox, syringes, nitrile gloves, sterile mask, mad labcoat, edagger, sleepy pen, sci hud eyepatch and megaphone

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 4]
```diff
@@ -163,6 +163,63 @@
...
+	new /obj/item/gun/syringe/syndicate(src)
+	new /obj/item/grenade/clusterbuster/metalfoam(src)
+	new /obj/item/grenade/clusterbuster/soap(src)
+	new /obj/item/storage/toolbox/syndicate(src)
+	new /obj/item/storage/box/syringes(src)
+	new /obj/item/clothing/suit/toggle/labcoat/mad(src)
+	new /obj/item/clothing/gloves/color/latex/nitrile(src)
+	new /obj/item/grenade/clusterbuster/spawner_spesscarp(src)
+	new /obj/item/pen/edagger(src)
+	new /datum/loadout_item/mask/job/surgical(src)
```
Comment (YakumoChen) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: wrong path for the mask

**Comment by Sethnov** [comment]
what i like is removing the elite hardsuit.

**Comment by hugmeorelse** [comment]
bump

**Pull request merged**